pkg/common/keys.go

```diff
@@ -242,12 +238,8 @@ func (rl *redisKeys) WorkspaceAuthorizedToken(token string) string {
 }
 
 // WorkerPool keys
-func (rk *redisKeys) WorkerPoolLock(poolName string) string {
```

Comment:
was this originally for loading the pools or something?

Reply:
I believe so - when we shared the worker pool configs between the gateways in v1's agent setup.

Reply:
👍